### PR TITLE
"Zeit-now" CLI to "zeit-vercel" CLI

### PR DESCRIPTION
**Pull request by andy-mishechkin**
# Description
According to https://github.com/actions/virtual-environments/issues/882 installing of the Zeit now has been replaced to installing of the Zeit Vercel. For further using Now command symlink is created to Vercel command.
Successful builds:
https://github.visualstudio.com/virtual-environments/_build/results?buildId=70312&view=results
https://github.visualstudio.com/virtual-environments/_build/results?buildId=70313&view=results

## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [x] Changes are tested and related VM images are successfully generated

### CODE REVIEW

**Comment by miketimofeev** [comment]
Rename file name `zeit-now.sh` -> `zeit-vercel`?

**Comment by AlenaSviridenko** [comment]
please, also fix PR title

File: images/linux/scripts/installers/zeit-now.sh [hunk 1]
```diff
@@ -7,16 +7,19 @@
...
-# Install the Zeit Now CLI
-npm i -g now
+# Install the Zeit Vercel CLI
+npm i -g vercel
 
 # Validate the installation
 echo "Validate the installation"
-if ! command -v now; then
-    echo "Zeit Now CLI was not installed"
+if ! command -v vercel; then
```
Comment (maxim-lobanov) [inline images/linux/scripts/installers/zeit-now.sh:15]: I think we should test `now` command too.

File: images/linux/scripts/installers/zeit-now.sh [hunk 2]
```diff
@@ -7,16 +7,25 @@
 # Source the helpers for use with the script
 source $HELPER_SCRIPTS/document.sh
 
-# Install the Zeit Now CLI
-npm i -g now
+# Install the Zeit Vercel CLI
```
Comment (maxim-lobanov) [inline images/linux/scripts/installers/zeit-now.sh:10]: Could you please also rename this file `zeit-now.sh` -> `zeit-vercel.sh`?

**Pull request merged**